tigerpath/majors_and_certificates/majors/PHI_2018.json

```
@@ -15,143 +15,508 @@
+              "course_list": [
+                "PHI *",
+                "POL *",
+                "HIS *",
+              ]
+            }
           ]
         },
-        {
-          "name": "History of Philosophy",
-          "max_counted": 1,
-          "min_needed": 2,
-          "explanation": "",
-          "course_list": [
-            "PHI 200",
-            "PHI 205",
-            "PHI 300",
-            "PHI 301",
-            "PHI 302",
-            "PHI 303",
-            "PHI 304",
-            "PHI 306",
-            "PHI 332",
-            "PHI 333",
-            "PHI 335",
-            "PHI 338"
-          ]
-        }
-      ]
-    },
-    {
-      "name": "Interdisciplinary Options",
-      "max_counted": null,
-      "min_needed": null,
-      "explanation": "",
-      "req_list": [
         {
           "name": "Political Philosophy",
           "max_counted": 1,
           "min_needed": "ALL",
-          "explanation": "Political Philosophy. Senior concentrators doing their theses in political philosophy have the option of substituting for the usual distribution requirement (two courses in each of three areas plus two unrestricted courses) the following: two courses from among those listed under the Department of Politics as courses in political theory; two philosophy courses in the ethics and philosophy of value area; two philosophy courses in one other philosophy distribution area; and two philosophy courses unrestricted as to distribution area.",
-          "no_req": true
+          "explanation": "Senior concentrators doing their theses in political philosophy have the option of substituting for the usual distribution requirement (two courses in each of three areas plus two unrestricted courses) the following: two courses from among those listed under the Department of Politics as courses in political theory; two philosophy courses in the ethics and philosophy of value area; two philosophy courses in one other philosophy distribution area; and two philosophy courses unrestricted as to distribution area.",
+          "req_list": [
+            {
+              "name": "Political Theory",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "<i>Editor's Note: Some of these courses fall under this requirement but TigerPath does not verify this.</i>\n\nTwo courses from among those listed under the Department of Politics as courses in political theory.",
+              "course_list": [
+                "POL *"
+              ]

```

The department of politics lists it explicitly [here](https://politics.princeton.edu/undergraduate/complete-course-listing?field-of-study%5B22%5D=22) as well as in their [undergraduate announcement](https://ua.princeton.edu/academic-units/department-department-politics).
```suggestion
              "explanation": "Two courses from among those listed under the Department of Politics as courses in political theory.",
              "course_list": [
                "POL 210",
                "POL 301",
                "POL 302",
                "POL 303",
                "POL 304",
                "POL 305",
                "POL 306",
                "POL 307",
                "POL 308",
                "POL 309",
                "POL 313",
                "POL 315",
                "POL 316",
                "POL 321",
                "POL 403",
                "POL 410",
                "POL 411",
                "POL 412",
                "POL 413",
                "POL 416"
              ]
```